pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -39,6 +267,7 @@ class MWACorrFITS(UVData):
     read_mwa_corr_fits method on the UVData class.
     """
 
+    # @profile
```

Comment:
do we want to keep these commented in the code?

Reply:
nope :)